src/pages/index.js

```diff
@@ -107,13 +209,33 @@ const createCard = (data) => {
 // section instance
 const cardList = new Section(
   {
-    items: initialCards,
     renderer: (item) => {
       cardList.addItem(createCard(item));
     },
   },
   selectors.cardSection
 );
-cardList.renderItems(initialCards);
+
 //userinfo instance
-const userInfo = new UserInfo(profileTitle, profileDescription);
+const userInfo = new UserInfo(profileTitle, profileDescription, profilePicture);
+
+api.getInitialCards().then((cards) => {
+  cardList.renderItems(cards);
+});
```

Comment:
You need to `catch` possible errors at the very end of any server request.

**Please, correct it everywhere**